src/contrib/cluster/Akka.Cluster.Sharding/Shard.cs

```diff
@@ -1095,10 +1118,7 @@ private void TryGetLease(Lease lease)
             Log.Info("{0}: Acquiring lease {1}", _typeName, lease.Settings);
 
             var self = Self;
-            lease.Acquire(reason =>
-            {
-                self.Tell(new LeaseLost(reason));
-            }).ContinueWith(r =>
+            lease.Acquire(reason => { self.Tell(new LeaseLost(reason)); }).ContinueWith(r =>
```

Comment:
weird churn compared to the rest of the churn....

Reply:
I think that was just me auto-formatting the document when I was doing things like removing nested `switch` statements